content/en/docs/guides/backup-and-restore.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Back up and Restore
+linkTitle: Back up and Restore
+description: "How to back up and restore resources in cozystack cluster."
+weight: 40
+aliases:
+  - /docs/backup-and-restore
+---
+
+CozyStack uses [Velero](https://velero.io/docs/v1.16/) to manage Kubernetes resource backups and recovery, including volume snapshots.
+This guide explains how to configure Velero and perform backups and recovery with practical examples.
+
+
+## Prerequisites
+
+To back up data in Cozystack, you need an S3-compatible storage ready.
+
+We recommend using the Velero CLI: [https://velero.io/docs/v1.16/basic-install/#install-the-cli](https://velero.io/docs/v1.16/basic-install/#install-the-cli)
+
+## 1. Set up Storage Credentials and Configuration
+
+To enable backups, the first step is to provide Cozystack with access to an S3-compatible storage.
+
+All configuration settings are created as secrets in the `cozy-velero` namespace.
+
+### 1.1 Create a Secret with S3 Credentials
+
+Create a secret containing credentials for your S3-compatible storage where backups will be saved.
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: s3-credentials
+  namespace: cozy-velero
+type: Opaque
+stringData:
+  cloud: |
+    [default]
+    aws_access_key_id=<KEY>
+    aws_secret_access_key=<SECRET KEY>
+
+    services = seaweed-s3
+    [services seaweed-s3]
+    s3 =
+        endpoint_url = https://s3.tenant-name.cozystack.example.com
+```
+
+### 1.2 Configure Backup Storage Location
+
+This configuration defines the S3 bucket where Velero stores backups.
+For more information, see: https://velero.io/docs/v1.16/api-types/backupstoragelocation/.
+
+```yaml
+apiVersion: velero.io/v1
+kind: BackupStorageLocation
+metadata:
+  name: default
+  namespace: cozy-velero
+spec:
+  provider: aws
+  objectStorage:
+    bucket: <BUCKET NAME>
+  config:
+    checksumAlgorithm: ''
+    profile: "default"
+    s3ForcePathStyle: "true"
+    s3Url: https://s3.tenant-name.cozystack.example.com
+  credential:
+    name: s3-credentials
+    key: cloud
+```
+
+### 1.3 Configure Volume Snapshot Location
+
+Defines configuration for volume snapshots.
+
+```yaml
+apiVersion: velero.io/v1
+kind: VolumeSnapshotLocation
+metadata:
+  name: default
+  namespace: cozy-velero
+spec:
+  provider: aws
+  credential:
+    name: s3-credentials
+    key: cloud
+  config:
+    region: "us-west-2"
+    profile: "default"
+```
+
+For more information see: https://velero.io/docs/v1.16/api-types/volumesnapshotlocation/.
+
+
+## 2. Create Backups
+
+Once the storage is configured, you can create backups manually or set up a schedule.
+
+## 2.1. Create a Backup Manually
+
+To create a backup manually, apply the following resource to the cluster:
+
+```yaml
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  name: manual-backup
+  namespace: cozy-velero
+spec:
+  snapshotVolumes: true
+  includedNamespaces:
+    - tenant-backuppvc
+  labelSelector:
+    matchLabels:
+      app: test-pod
```

Comment:
So we're backing pu the PVC of `test-pod` app in `backuppvc` namespace.

`test-pod` is in fact the name of the application, not a pod, is that correct?